manifests/gemrc.pp

```diff
@@ -0,0 +1,29 @@
+class classroom::gemrc (
+  Boolean $offline = false,
+) {
+  # NOTE: this online version of the .gemrc should match the one in pltraining-bootstrap
+
+  if $::osfamily == 'windows' {
+    file { ['C:/Users/Administrator/.gemrc', 'C:/ProgramData/PuppetLabs/puppet/etc/.gemrc' ]:
+      ensure  => file,
+      owner   => 'Administrator',
+      group   => 'Administrators',
+      mode    => '0644',
```

Comment:
Do I remember there being some problem with assigning permission modes to files on Windows? Maybe it cause the files to become inaccessible? Ah, the style guide recommends against it: https://docs.puppet.com/puppet/5.1/style_guide.html#file-modes

Reply:
yes,  acl > file modes > nothing

unfortunately, we're not doing anything with ACLs yet, so I think this will have to do for now. There's a ticket somewhere to port all the file permissions in this module to ACLs on windows.

Reply:
I don't think this should cause problems. It did with the sshkey because it was not group readable. (if I understand it correctly)